canonical-websites/cn.ubuntu.com
# Remove kylin download links

## Done

Remove kylin download links

## QA

- Checkout branch
- `./run serve`
- Go to the [downloads page](http://0.0.0.0:8010/download/)
- See that kylin download links are no longer there

## Issue / Card

[Fixes #188](https://app.zenhub.com/workspace/o/canonical-websites/cn.ubuntu.com/issues/188)
